Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'dry-component', github: 'dryrb/dry-component', branch: 'master'
+gem 'transflow', github: 'timriley/transflow', branch: 'fix-step-bug'
```

Comment:
This is here because dry-pipeline changed and caused transflow to start failing. Our choices here are to merge https://github.com/solnic/transflow/pull/5 and release a new transflow, or leave this line in place until we've pulled everything else from this repo into their own gems :)